### PR TITLE
Update faq.md

### PR DESCRIPTION
**Pull request by flacoonb**
add ### Raspberry Touchpanel DSI simultaneously with HDMI

#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/andi34/photobooth/blob/dev/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "x" next to an item)

- [x] Documentation update
- [ ] Bug fix
- [ ] New feature
- [ ] Other, please explain:

#### What changes did you make? (Give an overview)
Add FAQ Raspberry Touchpanel DSI simultaneously with HDMI

### CODE REVIEW

File: faq/faq.md [hunk 1]
```diff
@@ -474,3 +474,72 @@ A USB drive / stick can be identified either by the USB stick label (e.g. `photo
...
+Type=Application
+Exec=xinput set-prop 6 --type=float 136 0.3478260869565217 0 0 0 0.55555555555556 0 0 0 1 ##Adapt command to own values
+Terminal=false
+```
+
+Adjust rights and ownership:
+
+```
+sudo chmod +x ~/.config/autostart/touch.desktop
+sudo chown pi:p ~/.config/autostart/touch.desktop
```
Comment (andi34) [inline faq/faq.md]: `pi:p` ?

File: faq/faq.md [hunk 2]
```diff
@@ -474,3 +474,72 @@ A USB drive / stick can be identified either by the USB stick label (e.g. `photo
...
+```
+
+If you want to use the touchscreen as photobooth and the second monitor for the gallery, enter the following in the autostart file and adjust the value 1920 to your own resolution if necessary:
```
Comment (andi34) [inline faq/faq.md]: I'd adjust the thx to the following (standalone slideshow not Gallery), maybe spilt the `nano` part and the entry (else user might so add `sudo nano /etc/xdg/lxsession/LXDE-pi/autostart` inside the Autostart script).

If you want to use the touchscreen as photobooth and the second monitor for the standalone slideshow,  open the autostart file and enter the following in the `@chromium-browser` entries (adjust the value `1920` to your own resolution if necessary):

**Review by andi34 — COMMENTED** [review]
Thanks! Found 2 nit-picks, maybe you can adjust them?

File: faq/faq.md (hunk 1 above)
Comment (flacoonb) [reply faq/faq.md]: fix

File: faq/faq.md (hunk 2 above)
Comment (flacoonb) [reply faq/faq.md]: text switched and nano split

File: faq/faq.md [hunk 3]
```diff
@@ -474,3 +474,73 @@ A USB drive / stick can be identified either by the USB stick label (e.g. `photo
...
+
+```
+xinput set-prop 6 --type=float 136 0.3478260869565217 0 0 0.55555555555556 0 0 0 1
+```
+
+Now unfortunately the settings are only valid for the current session. So create the following desktop startup file:
+
+```
+sudo nano ~/.config/autostart/touch.desktop
+
```
Comment (andi34) [inline faq/faq.md:523]: Split here too?

Comment (flacoonb) [reply faq/faq.md:523]: Done

**Comment by andi34** [comment]
Added https://github.com/andi34/photobooth/commit/bdb3f0e49939514bca0a4e03c1bb18a3f8f93ec1

**Pull request closed without merging**